src/converter.js

```diff
@@ -15,14 +15,22 @@ function conversions(comment) {
   const formattedConversions = ch.formatConversion(roundedConversions);
   
   return formattedConversions.reduce((memo, conversion) => {
-    const key = conversion['imperial']['number'] + conversion['imperial']['unit'];
+    function parse(item) {
+      let result = item[0];
+      if(item.length > 1){
+        result += '-'+item[1];
+      }
+      return result;
+    }
+    let key = parse(conversion['imperial']['numbers']);
```

Comment:
I think we can delete "parse", and instead use the built-in [Array.join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) function, so it becomes:

`const key = conversion['imperial']['numbers'].join(-) + conversion['imperial']['unit']`

If you feel like the line is too long, you can pull out `conversion['imperial']` into its own variable. But either way, I think it's clearer if variables are declared `const` as often as possible.

Reply:
Yeah, I’m aware about this. I plan to rewrite this anyway in a seperate PR, which would be to handle variable joint words. Was holding off improvements for this till then. Should I improve it for now?